flask_restplus/namespace.py

```diff
@@ -201,9 +201,10 @@ def expect(self, *inputs, **kwargs):
         :param bool validate: whether to perform validation or not
 
         '''
+        validate = kwargs.get('validate', None)
         expect = []
         params = {
-            'validate': kwargs.get('validate', None) or self._validate,
+            'validate': validate if validate is not None else self._validate,
```

Comment:
There is no need to separate out retrieving `validate` from `kwargs` as the default parameter for `dict.get` can use `self._validate`:
```python
kwargs.get('validate', self._validate)
```

Reply:
Oh, you are right ... much easier to read!